saleor/webhook/transport/asynchronous/transport.py

```diff
@@ -48,10 +53,37 @@
 OBSERVABILITY_QUEUE_NAME = "observability"
 
 
+def create_deliveries_for_subscriptions_deferred_payload(
```

Comment:
Question: as we don't make any subscription here (this is done by celery task), maybe worth to consider dropping `subscriptions` from here?
I mean something like `create_deliveries_for_deferred_events`? 🤔 
(If you don't like this, just mark it as resolved 😄 )